src/main/webapp/src/components/Phones.js

```diff
@@ -187,9 +205,11 @@ class Phones extends React.Component {
                     />
                     <div className="priceCart">
                         <div className="price">{phone.price} RUB</div>
-                        <a href="/#" className="cart">
-                            <ShoppingCartOutlined/>
-                        </a>
+                            <div className="cart">
+                               <span onClick={() => this.addToCart(index)}>
```

Comment:
you can pass phone directly here. not index

Reply:
+